### DISCUSSION TITLE
zh-cn/manual/newbie

### DISCUSSION THREAD

**Discussion by giscus** [MaaAssistantArknights/maa-website#173] [Comments]
# zh-cn/manual/newbie

新手上路 一句话教学！ 前置准备 确认系统版本 MAA 仅支持 Windows 10 / 11，旧版 Windows 请参阅置顶。 安装运行库 MAA 需要 VCRedist x64 和 .NET 8，请右键开始按钮打开终端，在终端内粘贴以下命令回车以进行安装。 更多信息参考。 下载正确的版本 MAA 官网一般会自动选择正确的版本架构，对于大多数阅读本...

https://docs.maa.plus/zh-cn/manual/newbie.html

**Comment by AzHe4377** [comment]
MACOS卡在正在链接是怎么办？需要调整什么设置吗？

游戏版本是Appstore下载的最新版本

MacOS版本是“Sonoma 版本14.6.1 (23G93)”

期待答复

**Reply by Daydreamer114** [reply]
https://maa.plus/docs/zh-cn/manual/device/macos.html

**Comment by RieMannLie** [comment]
你好，我想请问下用maa打新界园，可以正常开局选分队招募，但是开第一关之后不会自己放干员是怎么办？ 期待答复，谢谢！

**Reply by Daydreamer114** [reply]
请在[主仓库](https://github.com/MaaAssistantArknights/MaaAssistantArknights/issues)中提交 issue

**Comment by 1974967971lmh-prog** [comment]
支不支持手机端
